docs/src/pages/migration-guide.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Migrating to v0.21
+description: How to migrate projects from Astro v0.20.
+---
+
+Starting in v0.21, Astro is now built with [Vite].
+As a result, configurations written in `snowpack.config.mjs` should be moved into `astro.config.js`.
```

Comment:
Maybe we should keep the `.mjs` extension? Assuming they don’t have `"type": "module"`

Reply:
Yep, agree with that. `.mjs` is a better recommendation for the docs since it works in both cases. If they already understand the nuances of Node ESM, they can change it.

Reply:
Fixed in 47d95716acd644ba97eb6e71ef0724452cbd1c9f